LR_3/Частотный анализ букв в тексте.py

```diff
@@ -0,0 +1,80 @@
+def count_letters(text):
+    russian_alphabet = [
+        'а', 'б', 'в', 'г', 'д', 'е', 'ё', 'ж', 'з', 'и',
+        'й', 'к', 'л', 'м', 'н', 'о', 'п', 'р', 'с', 'т',
+        'у', 'ф', 'х', 'ц', 'ч', 'ш', 'щ', 'ъ', 'ы', 'ь',
+        'э', 'ю', 'я'
+    ]
+
+    letter_counts = {letter: 0 for letter in russian_alphabet}

```

Comment:
В данном случае можно задать пустой словарь
```suggestion
    letter_counts = {}

```